packages/server/app/Controllers/Http/AuthenticationController.ts

```diff
@@ -12,22 +15,106 @@ export const SignInRequestBody = z.object({
 });
 export type SignInRequestBody = z.infer<typeof SignInRequestBody>;
 
+// Inspired by https://stackoverflow.com/questions/5142103/regex-to-validate-password-strength
+// Password has to have 2 caps letter one special char 2 digits and 3 lowercase char and minimal length has to be 8
+// Valid password examples: ':net66LTW', 'RN4k`d8he9k.'
+// Invalid password examples: 'bestpasswor@1', 'abcqwerty', 'ABCWE'
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏👌